lib/rules/id-length.js

```diff
@@ -40,6 +42,10 @@ module.exports = function(context) {
             return parent.id === node;
         },
         "Property": function(parent, node) {
+            if (!properties) {
```

Comment:
Oh, you're right, I can simplify to that expression. Would you rather I do that or prefer the second approach?